### DISCUSSION TITLE
mpv播放疑似不能完全调用4090

### DISCUSSION THREAD

**Discussion by Galahahad** [hooke007/mpv_PlayKit#396] [Q&A 问答]
非常奇怪，之前用3080在一定超频的基础上可以做到4k影片实时补帧，基本跑满cuda占用；
换4090后反而不行了，占用率一直在70%左右徘徊，换了几个版本的mpv_lazy都不行；
甚至有的4k的电影播放都有卡顿，在更改video-sync等参数后可以正常播放，但是还是不能补帧。
所以前来询问。

**Comment by Galahahad** [comment]
又试了一下，4090可以实时地把1080p的视频先补2倍帧再超分到4k；
但是反过来先超分再补帧就吃不消了。

**Reply by hooke007** [reply]
顺序当然影响性能

**Reply by Galahahad** [reply]
但是以前3080的时候4k视频能够实时补帧，刚好吃满；
4090现在不能实时补帧，不知道大佬有没有什么思路。

**Reply by hooke007** [reply]
![Snipaste_2023-12-27_04-39-25](https://github.com/hooke007/MPV_lazy/assets/41094733/158fa009-4d7d-46df-856d-be8d9b450638)
我只有4080，先1080p升4k(esrgan janaiv2L1)再rife_nv倍帧，理论上也只有36fps（放在mpv渲染管线里只会更低），要么你记错了哪里的流程。要么是我的4080有问题~~

**Reply by hooke007** [reply]
我在这里的建议是，不要压榨少的可怜的tensorcore单元，用rife_std合理利用硬件资源。

![Snipaste_2023-12-27_19-08-16](https://github.com/hooke007/MPV_lazy/assets/41094733/90864521-7443-4f36-a486-30fc100e4503)

**Reply by Galahahad** [reply]
谢谢大佬回答
